source/Tasks/CompressModulesWithChecksum.build.ps1

```diff
@@ -67,6 +67,9 @@ task CompressModulesWithChecksum {
         $modulesWithDscResources = $allModules | Where-Object Name -In $modulesWithDscResources
         #TODO: be more selective and maybe check based on the MOFs (but that's a lot of MOF to parse)
 
+        # As outlined here: https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.zipfile?view=net-6.0#remarks
+        Add-Type -AssemblyName System.IO.Compression.FileSystem
```

Comment:
as in other PR. Why not in module manifest?

Reply:
AFAIK, RequiredAssemblies makes sure that an an assembly exists but does not import it.

Reply:
As per the PS Doc:
https://docs.microsoft.com/en-us/powershell/scripting/developer/module/how-to-write-a-powershell-module-manifest?view=powershell-7.2

> Assemblies that must be loaded prior to importing this module. Specifies the assembly (.dll) file names that the module requires.
PowerShell loads the specified assemblies before updating types or formats, importing nested modules, or importing the module file that is specified in the value of the RootModule key. Use this parameter to list all the assemblies that the module requires.

Reply:
Given it's a task (and not a function), I'm ok to leave it as "Add-Type" because it's a bit weird behaviour.
I think we should `also` try to add it to the module manifest, but that'd be less urgent.